.rubocop_todo.yml

```diff
@@ -1,12 +1,12 @@
 # This configuration was generated by
 # `rubocop --auto-gen-config --auto-gen-only-exclude --exclude-limit 1400`
-# on 2022-08-29 05:26:26 UTC using RuboCop version 1.35.1.
+# on 2023-04-01 00:21:28 UTC using RuboCop version 1.47.0.
```

Comment:
Doesn't matter now, but it would have been good to regenerate the file in a separate commit, before making these changes. Then the changes can be viewed in isolation.

Reply:
That makes sense. I'll remember for next time. Thanks!